templates/tmate-ssh-server.service.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Docker instance to serve tmate-ssh-server.
+After=network.target
+
+[Service]
+User=ubuntu
+Group=ubuntu
+WorkingDirectory={{ WORKDIR }}
+ExecStart=sudo docker run --net=host --cap-add SYS_ADMIN -v {{ KEYS_DIR }}:/keys \
+    --env SSH_KEYS_PATH=/keys tmate/tmate-ssh-server -h 0.0.0.0 -p {{ PORT }} -k /keys
```

Comment:
Should this use the `-A` option?

Reply:
I would like to follow this up with another PR with the -A option. The Docker image provided upstream is heavily outdated (built 3 years ago) and does not contain the -A option.
In order to do so, I need to upload the newer Docker image (I'm thinking of using ghcr.io, I already have the rockcraft.yaml) and have an action to build, upload and tag the image.
What do you think?

Reply:
To be in a followup PR.